Autisticium

```diff
@@ -0,0 +1,10 @@
+/datum/chemical_reaction/autisticium
```

Comment:
okay this is my first time coding so im gonna need so much help to make this work. Okay thanks